install-dependencies.sh

```diff
@@ -165,6 +165,7 @@ declare -A pip_packages=(
     [pykmip]=""
     [universalasync]=""
     [boto3-stubs[dynamodb]]=""
```

Comment:
Please fix the git author/committer email of the patches to use your scylladb email.